.github/workflows/e2e-nvidia-a10g-x4.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         with:
           mode: start
           github-token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
-          ec2-image-id: ami-00c51d9c1374eda97
+          ec2-image-id: ami-03f6686697972e40a
```

Comment:
I'm clueless about how AMIs are picked for CI, so the question is probably ignorant but: why is the AMI changed here and what does the change represent? Is the motivation to sync with the ilab-runner version regardless what is there, or does the change represent something material needed for the CI?

Reply:
My assumption is that this is related to @n1hility's change to the cloud config scripting here: https://github.com/instructlab/instructlab/pull/1818

Reply:
That is correct its a material change of need so that we have an aligned set of drivers that mirror the ai-lab bootc image and meet the deps of the vllm release.

Reply:
The AMI change here is relevant because it has all of the cuda drivers properly installed to build vLLM. This is happening when we run `pip install -r requirements-vllm-cuda.txt`. 

Without the AMI change, that `pip install` would error out.

Reply:
Thank you for the explanation. You rock.